src/components/editableTable/EditableTable.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Table from '@material-ui/core/Table';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import TableCell from '@material-ui/core/TableCell';
+import TableBody from '@material-ui/core/TableBody';
+import Button from '@material-ui/core/Button';
+import AddIcon from '@material-ui/icons/Add';
+import EditableTableRow from './EditableTableRow';
+
+export default function EditableTable({ columns, rows, saveRow, createRow, editable, ...rest }) {
+    const [showNewRow, setShowNewRow] = useState(false);
+
+    return (
+        <Table size="small">
+            <TableHead>
+                <TableRow>
+                    {columns.map(column => (
+                        <TableCell key={column.id}>{column.title}</TableCell>
+                    ))}
+                    <TableCell />
+                    <TableCell />
```

Comment:
Are these empty cells just spacing so that the buttons have columns?

Reply:
Yeah, just so there line underneath the column names extends all the way out